hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -156,6 +158,28 @@ def collect_alembic_caches(self, parent_item, project_root):
                 cache_path
             )
 
+    def collect_alembic_caches_geometry(self, parent_item):
+        """
+        Creates items for alembic caches
+
+        :param parent_item: Parent Item instance
+        """
+
+        session_item = parent_item.create_item(
+            "maya.session.geometry",
+            "Alembic Cache",
+            "scenea.abc"
```

Comment:
My preference would be not to make this look like a filename. The filename will be driven by the publish_template. So maybe this should just be "All Session Geometry"?